### ISSUE TITLE
Пуля до 10, но не закрывает других игроков

### ISSUE THREAD

**Issue by kinguru** [danpetruk/preferans-calculator#1]
![Screenshot_20241101-163952](https://github.com/user-attachments/assets/99ed0547-c5e1-4f9a-8f8a-b46b4dc7af76)

Также похоже гора некорректно ставится

**Comment by danpetruk** [comment]
Да, действительно, фичи закрытия других игроков нет, для упрощения сделано так, что пуля продолжает писаться выше лимита, и калькулятор говорит о конце игры, когда сумма пуль >=лимит на пулю * кол-во игроков. С точки зрения итоговых очков математически результат выходит тем же
